### PR TITLE
README: Modify IoTivity README

### PR DESCRIPTION
**Pull request by heejin-kim**
To clarity the building steps, modify IoTivity README.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/iotivity/README.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
+~/TIZENRT_BASEDIR$ cd os/tools
+~/TIZENRT_BASEDIR/os/tools$ ./configure.sh <board>/iotivity
 ```
-
-Finally, initiate build by make from $TIZENRT_BASEDIR/os
+#### 2. not using IoTivity configuration
+Configuration can be modified through 'make menuconfig' from $TIZENRT_BASEDIR/os.  
 ```bash
-make
+~/TIZENRT_BASEDIR/os/tools$ cd ..
```
Comment (sunghan-chang) [inline external/iotivity/README.md]:
```bash
~/TIZENRT_BASEDIR$ cd os
~/TIZENRT_BASEDIR/os$ make menuconfig
```

File: external/iotivity/README.md [hunk 2]
```diff
@@ -1,32 +1,36 @@
 # IoTivity
 
-IoTivity is an open source software framework enabling seamless device-to-device and device-to-cloud connectivity to address the emerging needs of the Internet of Things.
+IoTivity is an open source software framework enabling seamless device-to-device and device-to-cloud connectivity to address the emerging needs of the Internet of Things.
```
[the remark this answers is not in the document]
Comment (heejin-kim) [reply external/iotivity/README.md]: To give a line btw the explanation and official web site. I am checking that again, there is empty line btw them. So I will revise just like so.
IoTivity is ~~. (two spaces)
Official page ~~.
Also give a hyperlink.

File: external/iotivity/README.md [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
 ```bash
-cd os/tools
-./configure.sh artik053/iotivity
+sudo apt-get install scons
 ```
+For more information refer to : http://www.scons.org/doc/production/HTML/scons-user.html#chap-build-install
 
-Configuration can be modified through make menuconfig from $TIZENRT_BASEDIR/os.
-You can enable security and cloud features for IoTivity.
+### Build with IoTivity function
```
[the remark this answers is not in the document]
Comment (heejin-kim) [reply external/iotivity/README.md]: I see

File: external/iotivity/README.md [hunk 4]
```diff
@@ -1,32 +1,36 @@
...
+Use IoTivity configuration for using IoTivity function.  
 ```bash
-cd ..
-make menuconfig
+~/TIZENRT_BASEDIR$ cd os/tools
+~/TIZENRT_BASEDIR/os/tools$ ./configure.sh <board>/iotivity
 ```
-
-Finally, initiate build by make from $TIZENRT_BASEDIR/os
+#### 2. not using IoTivity configuration
```
[the remark this answers is not in the document]
Comment (heejin-kim) [reply external/iotivity/README.md]: fixed

File: external/iotivity/README.md [hunk 5]
```diff
@@ -1,32 +1,36 @@
...
-cd os/tools
-./configure.sh artik053/iotivity
+sudo apt-get install scons
 ```
+For more information refer to : http://www.scons.org/doc/production/HTML/scons-user.html#chap-build-install
 
-Configuration can be modified through make menuconfig from $TIZENRT_BASEDIR/os.
-You can enable security and cloud features for IoTivity.
+### Build with IoTivity function
+#### 1. using IoTivity configration
```
[the remark this answers is not in the document]
Comment (heejin-kim) [reply external/iotivity/README.md]: fixed

File: external/iotivity/README.md (hunk 1 above)
Comment (heejin-kim) [reply external/iotivity/README.md]: fixed

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**